package.json

```diff
@@ -13,16 +13,24 @@
 		"pretty-quick": "pretty-quick --staged --pattern \"**/*.{scss,ts,js,json,yml}\""
 	},
 	"nohoist": [
-		"./@microsoft/atlas-css"
+		"**/@microsoft/atlas-css/"
 	],
```

Comment:
what's the nohoist for?

Reply:
added nohoist to ensure that during dev those files would exist in the same place they would after npm installing atlas-css